src/Lorentz/Contracts/RegistryDAO.hs

```diff
@@ -172,11 +173,52 @@ decisionLambda = do
     )
   nil
 
-config ::
+
+data CustomParam k v
+  = LookupRegistry (Lorentz.View k (k, Maybe v))
+  deriving stock (Generic, Show)
+  deriving anyclass (IsoValue, HasAnnotation)
+
+customEntrypointsHandler ::
+  forall s store k v.
+     ( IsoRegistryDaoProposalMetadata k v
+     , NiceParameter (k, v)
+     , DAO.StorageC store (RegistryDaoContractExtra k v) (RegistryDaoProposalMetadata k v)
+     ) => CustomParam k v : store : s
+    :-> List Operation : store : s
```

Comment:
Nitpick: please apply formatting to this function signature that is similar to the ones that already exist in this file. It's a bit inconsistent already as it seems there are two styles, so let's not add a third one.

Reply:
Have fixed the formatting.